src/shared/components/person/settings.tsx

```diff
@@ -1044,6 +1062,14 @@ export class Settings extends Component<any, SettingsState> {
     );
   }
 
+  handleOpenInNewTab(i: Settings, event: any) {
```

Comment:
You can define this at the module level instead of as a class method.

Reply:
Can you elaborate ? All of the other settings use instance level methods to mutate the state. I could extract it outside of the class but consistency has to be better right ?

Reply:
It's fine to leave it the way it is.